system_tests/noxfile.py

```diff
@@ -350,6 +350,20 @@ def external_accounts(session):
     )
 
 
+@nox.session(python=PYTHON_VERSIONS_SYNC)
+def downscoping(session):
+    session.install(
+        *TEST_DEPENDENCIES_SYNC,
+        "google-auth",
+        "google-cloud-storage",
+    )
+    default(
+        session,
+        "system_tests_sync/test_downscoping.py",
+        *session.posargs,
+    )
+
+
```

Comment:
Do we need to add a parallel `async` version of this test?

Reply:
I am not sure about the `async` project and effort. My team is not involved in that. I would rather not make this a blocking requirement for this project. I will get more information on that and how to resolve that long term.